sdk/storage/platform-matrix.json

```diff
@@ -15,6 +15,24 @@
       "Agent": { "macOS-10.15": { "OSVmImage": "macOS-10.15", "Pool": "Azure Pipelines" } },
       "JavaTestVersion": "1.11",
       "AZURE_TEST_HTTP_CLIENTS": "okhttp"
+    },
```

Comment:
The include array supports matrix definitions, so if you want you could remove one of these and simplify the other to:

```
    {
      "Agent": {
        "ubuntu-20.04": { "OSVmImage": "MMSUbuntu20.04", "Pool": "azsdk-pool-mms-ubuntu-2004-general" }
      },
      "JavaTestVersion": ["1.8", "1.11"],
      "AZURE_TEST_HTTP_CLIENTS": "netty",
      "StorageRunStressScenarios" : "true",
      "AdditionalOptions": "-DskipTests"
    },
```